app/src/main/java/fr/free/nrw/commons/upload/UploadActivity.java

```diff
@@ -103,7 +105,9 @@
     private Place place;
     private List<UploadableFile> uploadableFiles = Collections.emptyList();
     private int currentSelectedPosition = 0;
-    // Checks for if multiple files selected
+    /*
```

Comment:
Needs one more star here and another on the next line :-)
Must look like this:
```
/**
 * Example javadoc
 */
 ```
 Thanks!

Reply:
Sorry about that!